java-persistence/src/main/java/com/whitecloak/training/inventory/controller/request/UserForm.java

```diff
@@ -0,0 +1,27 @@
+package com.whitecloak.training.inventory.controller.request;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize
+public class UserForm {
+
+    private Long ownerId;
```

Comment:
Why are you asking for the `ownerId` when creating a user? Isn't it supposed to be filled after saving?